requirements.txt

```diff
@@ -1,20 +1,20 @@
-Flask==1.1.1
-Jinja2==2.11.3
-itsdangerous==1.1.0
+Flask==2.2
+Jinja2==3.0
+itsdangerous==2.0
 black==19.10b0
-click==6.7
-MarkupSafe==1.1.1
+click==8.0
```

Comment:
If this PR passes all the tests and doesn't introduce any dependency conflicts, then we can probably remove the `black==19.10b0` dependency above as well.

That was only added today to force `click` to keep being an older version, because newer click was causing test failures.